csrankings-c.csv

```diff
@@ -487,6 +487,7 @@ Chen-Ching Liu,Washington State University,https://school.eecs.wsu.edu/people/fa
 Chen-Nee Chuah,Univ. of California - Davis,https://www.ece.ucdavis.edu/~chuah/rubinet/people/chuah/bio.html,bZNRLNAAAAAJ
 Chenfanfu Jiang,University of Pennsylvania,http://www.seas.upenn.edu/~cffjiang,jTivVMEAAAAJ
 Cheng Chang Lu,Kent State University,http://www.cs.kent.edu/~lucc,NOSCHOLARPAGE
+Chengchao Shen,Central South University,https://faculty.csu.edu.cn/shenchengchao/zh_CN/index.htm,lQ8tqZ4AAAAJ

```

Comment:
This faculty member is listed as being a Master supervisor, not a PhD supervisor. Only faculty who can solely supervise PhDs in Computer Science are eligible for inclusion.

Reply:
Thanks for your relpy. We have removed any faculty who are unable to solely supervise PhD students according to your request.